.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     es2020: true,
     'jest/globals': true,
     node: true,
+    jasmine: true,
```

Comment:
👀

Reply:
`fail()` is method provided by `jasmine`. `jasmine` is a testing library on which `jest` is built. So by adding this line, I don't get a linting error telling me `fail` does not exists

Reply:
javascript-dependency-hell! Thanks for the explanation! I didn't know jest was based on jasmine!